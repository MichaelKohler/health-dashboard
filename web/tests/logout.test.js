/* global it, expect, jest */
/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Logout from '../src/logout.jsx';

jest.mock('react-redux', () => ({
  useDispatch: () => jest.fn(),
}));

it('renders correctly', () => {
  const wrapper = shallow(<Logout/>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

/* eslint-enable react/jsx-filename-extension */
