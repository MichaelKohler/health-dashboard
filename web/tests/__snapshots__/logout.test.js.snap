// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <div
    className="makeStyles-appBarSpacer-1"
  />
  <div
    className="makeStyles-paper-3"
  >
    <div
      className="MuiAvatar-root makeStyles-avatar-7 MuiAvatar-colorDefault"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <g
          fill="none"
        >
          <path
            d="M0 0h24v24H0V0z"
          />
          <path
            d="M0 0h24v24H0V0z"
            opacity=".87"
          />
        </g>
        <path
          d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
        />
      </svg>
    </div>
    <h1
      className="MuiTypography-root MuiTypography-h5"
    >
      Logout
    </h1>
    <form
      className="makeStyles-form-5"
      noValidate={true}
      onSubmit={[Function]}
    >
      <p>
        Are you sure you want to log out?
      </p>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-submit-6 MuiButton-containedSecondary MuiButton-fullWidth"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Log out
        </span>
      </button>
    </form>
  </div>
</main>
`;
